packages/client/src/styles/globals.css

```
@@ -1752,21 +1756,19 @@ body {
-  top: 50%;
   border-radius: 999px;
-  background: currentColor;
-  transform: translate(-50%, -50%);
 }
 
 .mari-home-starfield__star::before {
-  width: 100%;
-  height: 1px;
+  inset: 0;
+  background:
+    radial-gradient(circle, var(--foreground) 0 12%, currentColor 28%, transparent 72%),
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)